scapy/automaton.py

```diff
@@ -698,7 +698,10 @@ class CommandMessage(AutomatonException):
     # Services
     def debug(self, lvl, msg):
         if self.debug_level >= lvl:
-            log_interactive.debug(msg)
+            if conf.interactive:
```

Comment:
Is there a reason why we don't use logging for this?

Reply:
I wasn't sure which logger to use. The interactive one won't work out of a shell

Reply:
You need to configure the logger
```
import logging
logger = logging.getLogger("scapy")
logger.setLevel(logging.INFO)
logger.addHandler(logging.StreamHandler())
from scapy.all import * 
log_runtime.critical('error')
```

Reply:
In any case that's only used during the debug. It's super useful to debug the tls tests..
Where would we instantiace the logger otherwise.. In UTScapy?

Reply:
Yes, that will be the better option.